src/webgpu/shader/execution/expression/expression.ts

```diff
@@ -562,67 +562,92 @@ function packScalarsToVector(
   };
 }
 
+/** Indicates how strictly should intervals be checked. */
+export type IntervalCheck =
+  | 'f32' // Expected to be f32 finite
+  | 'none'; // No checks applied
```

Comment:
unfortunately, I'm still not able to look at `generateU32ToVectorCases(fullU32Range(), 'none', unpack4x8unormInterval);` and know what `'none'` means without looking up the parameter.

`'f32-only' | 'unfiltered'` or something would be much clearer when just reading the code that makes a call.

I don't want to block the change though, so feel free to land. We can always tidy this stuff up later.

Reply:
Done, hitting the patch with sed enough times made the change pretty mechanical